SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLUnsubscribeVehicleDataSpec.m

```diff
@@ -110,7 +112,9 @@
                                                            SDLRPCParameterNameSteeringWheelAngle:@YES,
                                                            SDLRPCParameterNameTirePressure:@YES,
                                                            SDLRPCParameterNameTurnSignal:@YES,
-                                                           SDLRPCParameterNameWiperStatus:@YES},
+                                                           SDLRPCParameterNameWiperStatus:@YES,
+                                                           @"stabilityControlsStatus":@YES
```

Comment:
`SDLRPCParameterNameStabilityControlsStatus `

Reply:
If I replace the strings with constants declared in another file how can I be sure the constants are correct?
In such case the test will test nothing will it not?